Created a base pages setup

RJaliliSTFC opened the pull request:
Creating a foundation to work on
Imported old images and plain text

RJaliliSTFC committed:
Created a base pages setup
Creating a foundation to work on
Changed cloud-support@gridpp to @stfc

RJaliliSTFC committed:
Created a base pages setup
Creating a foundation to work on

meoflynn commented on Templates/sla.html:
```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% block title %}SLA{% endblock %}
+{% block content %}
+<p>The SCD Cloud is a service developed for use within Scientific Computing and the wider STFC for developing and supporting projects. SCD Cloud usage must comply with the <a href="https://cloud.stfc.ac.uk/tos">Terms of Service</a>. If you have any questions regarding the following, please contact <a href="mailto:cloud-support@gridpp.rl.ac.uk">cloud-support@gridpp.rl.ac.uk</a></p>
```
Just need to update the cloud support email to `cloud-support@stfc.ac.uk`

meoflynn commented on Templates/tos.html:
```diff
@@ -0,0 +1,18 @@
+    <li>You UNDERSTAND that, whilst best effort is made to provide a stable platform, VMs may be subject to interruption and/or data loss at any time without notice (this is a development platform and should not be used for production services)</li>
+    <li>With respect to any SOFTWARE that you install you MUST ensure that all applicable license and terms and conditions of use are met </li>
+    <li>You MUST NOT use the service in any way related to providing any commercial service or running a private business</li>
+    <li>You MUST report any suspected or actual security incident or other misuse of the VM immediately to <a href="mailto:cloud-support@helpdesk.gridpp.rl.ac.uk">cloud-support@helpdesk.gridpp.rl.ac.uk</a> and notify the appropriate STFC security contact by following the procedure at <a href="https://staff.stfc.ac.uk/core/security/information/Pages/Incidents.aspx">https://staff.stfc.ac.uk/core/se ...
```
Same as above (`cloud-support@stfc.ac.uk`)

meoflynn commented on cloud.py:
```diff
@@ -0,0 +1,40 @@
+from flask import Flask, render_template, request, url_for, redirect, session
+
+app = Flask(__name__)
+app.secret_key = b'r4nd0ms3cr3tk3y'
```
Please replace it with a placeholder to make it clearer that it's just a placeholder for a key

meoflynn commented on templates/sla.html:
```diff
@@ -0,0 +1,34 @@
+<p>The SCD Cloud is a service developed for use within Scientific Computing and the wider STFC for developing and supporting projects. SCD Cloud usage must comply with the <a href="https://cloud.stfc.ac.uk/tos">Terms of Service</a>. If you have any questions regarding the following, please contact <a href="mailto:cloud-support@gridpp.rl.ac.uk">cloud-support@gridpp.rl.ac.uk</a></p>
```
Just need to update the email address here too (`cloud-support@stfc.ac.uk`)

meoflynn commented on templates/tos.html:
```diff
@@ -0,0 +1,14 @@
+    <li>You UNDERSTAND that, whilst best effort is made to provide a stable platform, VMs may be subject to interruption and/or data loss at any time without notice (this is a development platform and should not be used for production services)</li>
+    <li>With respect to any SOFTWARE that you install you MUST ensure that all applicable license and terms and conditions of use are met </li>
+    <li>You MUST NOT use the service in any way related to providing any commercial service or running a private business</li>
+    <li>You MUST report any suspected or actual security incident or other misuse of the VM immediately to <a href="mailto:cloud-support@helpdesk.gridpp.rl.ac.uk">cloud-support@helpdesk.gridpp.rl.ac.uk</a> and notify the appropriate STFC security contact by following the procedure at <a href="https://staff.stfc.ac.uk/core/security/information/Pages/Incidents.aspx">https://staff.stfc.ac.uk/core/se ...
```
Same as above

RJaliliSTFC committed:
Merge branch 'file_structure' of github.com:stfc/cloud into file_structure

meoflynn reviewed (changes requested):
Looks good to me, just some minor changes needed to update the email addresses in the html files

RJaliliSTFC committed:
Fixed comments
changed the support email
changed secret key to be a placeholder
moved files int templates from Templates

meoflynn commented on cloud.py:
```diff
@@ -0,0 +1,40 @@
+def sla():
+    return "Service Level Agreement Page <h1>SLA<h1>"
+
+# @app.route("/faqs")
+# def faqs():
+#     return "Frequently Asked Questions Page <h1>FAQ<h1>"
+
+# @app.route("/login", methods=["POST", "GET"])
+# def login():
+#     if request.method == "POST":
+#         user = request.form["username"]
+#         session["user"] = user
+#         return redirect(url_for("user"))
+#     else:
+#         return render_template("loginpage.html")
+
+# @app.route("/user")
+# def user():
+#     if "user" in session:
+#         user = session["user"]
+#         return render_template("testpage.html", passdata=user)
+#     else:
+#         return redirect(url_for("login"))
+
```
Has this page been removed or is it to be added in a future PR and just commented out for now?

RJaliliSTFC replied:
For future PR and commented out for now
